src/network/__init__.py

```diff
@@ -4,19 +4,19 @@
 
 try:
     from .announcethread import AnnounceThread
-    from .connectionpool import BMConnectionPool
+    from .connectionpool import BMConnectionPool as BMConnectionPool_cls
+    BMConnectionPool = BMConnectionPool_cls()
 except ImportError:
     AnnounceThread = None
     BMConnectionPool = None
 from .threads import StoppableThread
 
 
-__all__ = ["AnnounceThread", "BMConnectionPool", "StoppableThread"]
```

Comment:
I think we can use the object name here, i.e. `connectionpool` as I recommended above.